sgv2-docsapi/pom.xml

```diff
@@ -108,6 +108,16 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-opentelemetry-exporter-otlp</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.smallrye.reactive</groupId>
+      <artifactId>mutiny-rxjava3</artifactId>
+      <version>1.5.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.akarnokd</groupId>
+      <artifactId>rxjava3-extensions</artifactId>
+      <version>3.1.1</version>
+    </dependency>
```

Comment:
Maybe allude to the fact that this is needed because of ordered merge in `QueryExecutor`?

Although one can also comment the dependencies and try to recompile to see where it's used...

Reply:
Will add a comment.